CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ package to catch or to otherwise handle errors that you may encounter.
 ### Added
 
 - Added user doc describing new features related to the Cumulus dead letter archive.
+- **CUMULUS-2460**
+  - Adds `POST` /executions/search-by-granules for retriving workflow names from a given granule, list of granules, or granule query
```

Comment:
```suggestion
  - Adds `POST` /executions/search-by-granules for retrieving executions from a list of granules or granule query
```

Since the endpoint is returning full executions and not workflow names at this point